lib/google_crawler_web/templates/keyword/_keyword_filter.html.eex

```diff
@@ -0,0 +1,59 @@
+<div class="collapse keyword-filter" id="keywordFilter">
+  <div class="card card-body">
+    <form>
+      <div class="form-group row">
+        <div class="keyword-filter__item col-lg-4 col-6">
+          <label for="title">Keyword Title</label>
+          <input class="form-control" name="keyword_filter[title]" placeholder='"apple", "rocket"'
```

Comment:
Should we remove the double quote in `"apple", "rocket"`

Reply:
This one already fixed in the base branch. So it should be updated after rebasing 😁

Reply:
@andyduong1920 This PR is rebased, the placeholders are fixed 😁